.github/workflows/release.yaml

```diff
@@ -388,192 +388,193 @@ jobs:
+        with:
+          name: ${{ env.NIN_3DS_ARCHIVE_ARTIFACT }}
+          path: ${{ steps.create_nintendo_3ds_archive.outputs.archive_file }}
+          retention-days: 1
+
+  nds:
+    needs: [setup, docconvert]
+    name: Nintendo DS
+    runs-on: ubuntu-latest
+    container: skylyrac/blocksds:slim-latest
+    steps:
+      - name: Download Artifact with Configuration Information
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ env.CONFIG_ARTIFACT }}
+
+      - name: Extract Configuration Information and Store in Step Outputs
+        id: store_config
+        run: |
+          name=`sed -E -n -e 's/name= //p' $CONFIG_ARTIFACT_PATH`
+          echo "name=$name" >> $GITHUB_OUTPUT
+          prog=`sed -E -n -e 's/prog= //p' $CONFIG_ARTIFACT_PATH`
+          echo "prog=$prog" >> $GITHUB_OUTPUT
+          version=`sed -E -n -e 's/version= //p' $CONFIG_ARTIFACT_PATH`
+          echo "version=$version" >> $GITHUB_OUTPUT
+
+      - name: Clone Project
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Download Artifact with Converted Documents
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ env.DOC_ARTIFACT }}
+
+      - name: Extract Converted Documents
+        run: |
+          tar -xf ${{ env.DOC_ARTIFACT_PATH }}
+
+      - name: Create Nintendo DS Archive
+        id: create_nintendo_ds_archive
+        shell: bash
+        run: |
+          apt-get update && apt-get install -y zip
+          pushd src/
+          make -f Makefile.blocksds -j$(nproc)
+          popd
+          test -e src/${{ steps.store_config.outputs.prog }}.nds
+
+          archive_prefix=${{ steps.store_config.outputs.name }}-${{ steps.store_config.outputs.version }}
+          echo "archive_file=${archive_prefix}-nds.zip" >> $GITHUB_OUTPUT
+          mkdir -v ${{ steps.store_config.outputs.prog }}/
+          cp -v src/${{ steps.store_config.outputs.prog }}.nds ${archive_prefix}.nds
+          cp -rv lib ${{ steps.store_config.outputs.prog }}/
+          zip -r ${archive_prefix}-nds.zip ${archive_prefix}.nds ${{ steps.store_config.outputs.prog }}/
+
+      - name: Create Artifact with Nintendo DS Archive Path
+        run: |
+          echo archive_path= '${{ steps.create_nintendo_ds_archive.outputs.archive_file }}' > $NIN_DS_ARTIFACT_PATH
+
+      - name: Upload Artifact with Nintendo DS Archive Path
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.NIN_DS_ARTIFACT }}
+          path: ${{ env.NIN_DS_ARTIFACT_PATH }}
+          retention-days: 1
+
+      - name: Upload Nintendo DS Archive as Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.NIN_DS_ARCHIVE_ARTIFACT }}
+          path: ${{ steps.create_nintendo_ds_archive.outputs.archive_file }}
+          retention-days: 1
 
   release:
-    needs: [ setup, source, windows, mac ]
+#    needs: [ setup, source, windows, mac ]
```

Comment:
Delete the commented out lines 575 and 576.

Reply:
done https://github.com/angband/angband/pull/6469/changes/1f994486697d1526c18939c7b4b31a1d607ab8fd